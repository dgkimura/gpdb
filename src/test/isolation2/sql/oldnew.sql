1o: CREATE TABLE t(a int);
CREATE TABLE t(a int);

-- insert data on old cluster
1o: INSERT INTO t VALUES(1);

-- now perform pg_upgrade using like...
-- \! pg_upgrade --old-port=XXX --new-port=YYY ...

-- validate data on new cluster
SELECT * FROM t;
