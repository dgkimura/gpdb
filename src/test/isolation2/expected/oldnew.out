1o: CREATE TABLE t(a int);
CREATE
CREATE TABLE t(a int);
CREATE

-- insert data on old cluster
1o: INSERT INTO t VALUES(1);
INSERT 1

-- now perform pg_upgrade using like...
-- \! pg_upgrade --old-port=XXX --new-port=YYY ...

-- validate data on new cluster
SELECT * FROM t;
 a 
---
(0 rows)
